### DISCUSSION TITLE
Move documentation to Gitbook

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#144] [Ideas]
@Chacapamac I planning move Astroid documentation to Gitbook.com. I did it with Jollyany https://docs.jollyany.co/. And I need you help. 
Would you like to help me move and update documentation for Astroid on Gitbook? I will invite you to our team, so you can create page and upload content to Gitbook.

Kind Regards,
Sonny

**Comment by sonvnn** [comment]
@Chacapamac I was created a project for Astroid on Gitbook. https://docs.astroidframe.work/ You can check it here.

**Comment by Chacapamac** [comment]
I Sonny,

I’m trying hard to give time for the project, not a good period for me …

—————————————————————————————
Yes, I see your Jollyany Documentation, I understand that it will be easy to have all that at the same place as both Astroid & Jollyany are in a way the same.

I think I have something that can help?
I  make a copy of ALL Joomdev Documentation (with images and All) that can be a great start here.
—> https://drive.google.com/file/d/1H-A2y1Cn-bEyHw58oJGNa_KwCldUtv-e/view?usp=sharing <https://drive.google.com/file/d/1H-A2y1Cn-bEyHw58oJGNa_KwCldUtv-e/view?usp=sharing>

Everything is classified the same way it appear on original Joomdev documentation


Tell me if we can work with that and I can help you for  https://docs.astroidframe.work/ <https://docs.astroidframe.work/>



—————————————————————————————
Good News, 
Joomlashack is starting to help people with older Joomdev Astroid Templates. 

I’m helping a guy that use Joomdev Astroid with the VirtueMart Joomdev Template ServX that stop working on newer Joomla (he is in big trouble, multiple big UK commerce website done with it…)
— I already fix his problem using the last NEW Templaza Astroid and The NEW Template Zero
— Now Joomlashack sent him an updated version of ServX
— Will help him make it work


Good Day !


Patrick Toulze
Graf • Art Director — Marketing Consultant — Social Channels — SEO — Web Mobile

**Comment by sonvnn** [comment]
@Chacapamac Thank you for your help. In fact, Joomdev documentation is still available https://docs.joomdev.com/category/astroid-user-manual/ 
We just need move it to new place and update it for latest Astroid version. I will take a lot of time, may be one or two weeks. We still put it in pending queue because our team have full task now with some other projects. 

Please let me know when you have time with it. I am very grateful to you.

Kind Regards,
Sonny

**Comment by Chacapamac** [comment]
I Sonny,

Joomdev documentation can disappear at any time and, for the most, are really clear. That is why I keep a copy of it. It can be an excellent base for creating the new documentation. The only stuff missing is the NEW ASTROID features and possibilities that can be add or insert in the existing documentation. I think the documentation should be first centralize on the NEW ASTROID website.


I’m just a bit confused about how you want to proceed with two places for the documentation?
— Directly on the website —> https://astroidframe.work/support/documentations <https://astroidframe.work/support/documentations>
— Now the new place —> https://docs.astroidframe.work <https://docs.astroidframe.work/>

• Do you want to transport what is on the Astroid website to the “Doc” system?


Personally, this is what I was thinking of doing, tell me if it's OK with you?
I can start now, but my time is limited at the moment

— First, I start to revise each documentation article on the website, compare it with the original Joomdev documentation and create a concise clear article with video if available.
— Before creating the new article I will save a permanent Joomla version of the original for safekeeping (same with the new).
— When the article is finished, it can be transported to your Documentation platform

What do you think?








Something else, just a suggestion;
I think you need to explain somewhere the relation between Jollyany and Astroid

Example of possible confusion: 
On Astroid website you have a Template Providers submenu to Ressources where user can go to your Templaza templates. There they are told that the framework is Jollyany?

My toughts;
As Joomdev Astroid is now gone, you can appropriate the framework name and explain that Jollyany Templates use the Astroid Framework and add great add-on possibility to Astroid.
Something like that?

Also, I think that you should just cut the Ressources main menu and put Templates as a first level main menu. I don’t think it’s a good ideas to hide the Templates availability as a submenu. If you like the framework you want the templates that go with it. 



Good Day Sonny!

Patrick

**Comment by sonvnn** [comment]
Hi Patrick,

GitBook is modern documentation format and toolchain using Git and Markdown. You can learn about it at https://www.gitbook.com/
It will help users search & research Astroid easily. It's free for personal use. You can try it. 

At the moment our document is not friendly for user. It's very difficult to search & research information. UX is zero. That is the reason, I would like to move Astroid document to Gitbook system. I think it will help Astroid users a lot. The current document on website will be removed. Our member will start work with it in next 2 weeks after she finish her current task.

It's great if you can join and help us improve documentation content (Contents, Video tutorials, etc). 

For Jollyany. It's a child framework of Astroid. Jollyany = Astroid + UIKit Addon + TemPlaza membership function.
I noted in Jollyany description "Powerful [Astroid Framework](https://github.com/templaza/astroid-framework) latest version 2.5.17 made by [Bootstrap 5](https://getbootstrap.com/)" you can check it here https://themeforest.net/item/jollyany-responsive-multipurpose-joomla-template/8596818

In the resource menu. I would like encourage teams who use Astroid Framework. Now we have 2 Joomla template providers are TemPlaza and @Joomlaplates . With you suggestion, I moved it on top and rename to "Partners" https://astroidframe.work/partners

Please let me know if you have further questions.

Thanks & Best Regards,
Sonny

**Comment by Chacapamac** [comment]
> GitBook is modern documentation format and toolchain…..
> 
Ok, if I understand well your answer you really want to move (taking out) the documentation of the website and move it to Gitbook 


My first question is if Gitbook permit an obvious link back to the Astroid website?
My concern is that if the user is transport on another website/Gitbook, it decentralize the information making that a big UX mistake. You need to have a fluid experience and Gitbook must appear as if almost incorporated to the website.

> It's great if you can join and help us improve documentation content (Contents, Video tutorials, etc).
> 

For the documentation, I will start right now, but where you want me to work, the website or Gitbook?



> In the resource menu. I would like encourage teams who use Astroid Framework. Now we have 2 Joomla template providers are TemPlaza and @Joomlaplates <https://github.com/Joomlaplates> . With you suggestion, I moved it on top and rename to "Partners" https://astroidframe.work/partners <https://astroidframe.work/partners>Again, the following is a suggestion;
 “Partners” doesn’t mean “Templates” — People that comme to the Framework website for information doesn’t care about “Partners”, they care to see and access “Templates” made with that framework.
 


Good Day Sonny !

Patrick Toulze
